src/js/consts/StockMovementDirection.js

```diff
@@ -0,0 +1,4 @@
+export default {
+  INBOUND: 'INBOUND',
+  OUTBOUND: 'OUTBOUND',
+};
```

Comment:
To unify this "enum" with the backend part, please add "INTERNAL"
This is how it looks on the backend:
```
    INBOUND('Inbound'),
    OUTBOUND('Outbound'),
    INTERNAL('Internal')
```

Reply:
this is weird. I'm sure we've already had this enum on the frontend, but suddenly I can't find it anywhere.